src/test/java/org/folio/marc4ld/authority/field100/MarcToBibframeAuthorityConcept100IT.java

```diff
@@ -0,0 +1,207 @@
+package org.folio.marc4ld.authority.field100;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.folio.ld.dictionary.PredicateDictionary.FOCUS;
+import static org.folio.ld.dictionary.PredicateDictionary.MAP;
+import static org.folio.ld.dictionary.PredicateDictionary.SUB_FOCUS;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.CONCEPT;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.FAMILY;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.FORM;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.IDENTIFIER;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.ID_LCCN;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.PERSON;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.PLACE;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.TEMPORAL;
+import static org.folio.ld.dictionary.ResourceTypeDictionary.TOPIC;
+import static org.folio.marc4ld.mapper.test.TestUtil.loadResourceAsString;
+import static org.folio.marc4ld.mapper.test.TestUtil.validateEdge;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.commons.collections4.CollectionUtils;
+import org.folio.ld.dictionary.ResourceTypeDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.ld.dictionary.model.ResourceEdge;
+import org.folio.marc4ld.mapper.test.SpringTestConfig;
+import org.folio.marc4ld.mapper.test.TestUtil;
+import org.folio.marc4ld.service.marc2ld.authority.MarcAuthority2ldMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@EnableConfigurationProperties
+@SpringBootTest(classes = SpringTestConfig.class)
+class MarcToBibframeAuthorityConcept100IT {
+
+  @Autowired
+  private MarcAuthority2ldMapper marcAuthority2ldMapper;
+
+  @Test
+  void shouldMapField100() {
```

Comment:
Can we verify that the hashes of all the generated resources are correct?

Earlier, we had a problem in bib -> Graph transformation where Instance resource's hash was computed before setting all the properties of the resource. i,e, something as below

```
Resouce instance = new Resource();
setSomePropertiesOfResouce(instance);
reseource.setHash(hashService.hash(instance)); // hash is set before setting all properties.
setSomeMorePropertiesOfResouce(instance);
```

 In order to catch such errors, we later added a [validateId](https://github.com/FOLIO-EIS/lib-linked-data-marc4ld/blob/master/src/test/java/org/folio/marc4ld/mapper/Marc2BibframeMapperIT.java#L1113) method in `Marc2BibframeMapperIT.java`

I know this test is doesn't make much sense in the first look as we are making use of our application code in the test. The purpose of this validation is to ensure that resource's hash computed only after setting all its data. 

Can we add a test to ensure that the hash of all the generated resources are correct?

Reply:
It's not a problem. Id validation now is each of validation resource